lib/rets/client.rb

```diff
@@ -266,6 +266,9 @@ def retrieve_metadata_type(type)
                         "ID"     => "0"
                       })
       res.body
+    rescue Rets::InvalidRequest => e
+      puts e.to_s
+      return ""  
```

Comment:
What's the reasoning behind this? Rescuing errors and returning data makes me a little nervous that we'll hide errors and end up with corrupt data.